config/kubernetes/sig-release/teams.yaml

```diff
@@ -58,7 +27,7 @@ teams:
     - cheftako # Cloud Provider
     - childsb # Storage
     - chrigl # OpenStack
-    - cpanato # Branch Manager
```

Comment:
These hunks maybe got a 'git add -i' in the wrong commit?  Maybe just a nit though if the commits are to be squashed?